pygmt/tests/test_grd2cpt.py

```diff
@@ -5,17 +5,17 @@
 
 import pytest
 from pygmt import Figure, grd2cpt
-from pygmt.datasets import load_earth_relief
 from pygmt.exceptions import GMTInvalidInput
 from pygmt.helpers import GMTTempFile
+from pygmt.helpers.testing import load_static_earth_relief
 
 
 @pytest.fixture(scope="module", name="grid")
 def fixture_grid():
     """
     Load the grid data from the sample earth_relief file.
     """
-    return load_earth_relief()
+    return load_static_earth_relief()
```

Comment:
At line 30, the `frame="a2000"` no longer makes sense, because static_earth_relief is not a global grid.

Reply:
Changed